online-inference/tensorizer-isvc/model-download/model-download-job.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: model-download
+spec:
+  template:
+    spec:
+      containers:
+      - name: model-downloader
+        image: rtalaricw/model-download-gptj:v1.3
+        imagePullPolicy: IfNotPresent
+        volumeMounts:
+          - name: model-cache
+            mountPath: /mnt
+        resources:
+          requests:
+            cpu: 8
+            memory: 64Gi 
+            nvidia.com/gpu: 1
+          limits:
+            cpu: 8
+            memory: 64Gi 
+            nvidia.com/gpu: 1
```

Comment:
These resources are probably overkill for the model downloader. I don't think it should need a GPU at all.